src/components/Aframe/my-loader.js

```diff
@@ -574,7 +562,16 @@ AFRAME.registerComponent("myloader", {
     }
 
     if (oldData.volumeData !== this.data.volumeData) {
-      this.loadModel();
+		var parent_folder = this.data.volumeData.substr(0,this.data.volumeData.lastIndexOf("/")+1)
```

Comment:
the data folder is always constant, can that knowledge be added in a config object somewhere instead?

Reply:
As in harcoding the "./assets/models/" path?
I am fine with it (As long as we dont chage it in the near future)

Reply:
that could go in the `config.json`, but I think it's reasonable to assume for this application (and any forks thereof), the data will always be in that folder.

Reply:
config.json is added into the project as suggested. The config file contains data be loaded in the volume viewer.